homeassistant/components/satel_integra/__init__.py

```diff
@@ -45,20 +48,42 @@
 ZONE_SCHEMA = vol.Schema({
     vol.Required(CONF_ZONE_NAME): cv.string,
     vol.Optional(CONF_ZONE_TYPE, default=DEFAULT_ZONE_TYPE): cv.string})
+EDITABLE_OUTPUT_SCHEMA = vol.Schema({vol.Required(CONF_ZONE_NAME): cv.string})
+PARTITION_SCHEMA = vol.Schema(
+    {vol.Required(CONF_ZONE_NAME): cv.string,
+     vol.Optional(CONF_ARM_HOME_MODE, default=DEFAULT_CONF_ARM_HOME_MODE):
+     vol.In([1, 2, 3]),
+     }
+    )
+
+
+def is_alarm_code_necessary(value):
+    """Check if alarm code must be configured."""
+    if not isinstance(value, dict):
+        raise vol.Invalid('key dependencies require a dict')
+    if CONF_SWITCHABLE_OUTPUTS in value\
+       and value[CONF_SWITCHABLE_OUTPUTS]\
```

Comment:
You could combine to `value.get(CONF_SWITCHABLE_OUTPUTS)`

Reply:
True! Will address it in next push.